wiki/blocks/first-block.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Adding your first Block
+index: 1
+---
+# Adding your first Block
+
+Creating a Block is pretty similar to creating an Item, [read the corresponding article](../items/first-item) for more elaborate explanations.
+
+## Creating and registering the Block
+
+First we create the Block and store it in a field:
+
+```java
+public final Block EXAMPLE_BLOCK = new Block(new QuiltBlockSettings())
+```
+
+Then we register it in the `onInitialize()` function:
+
+```java
+Registry.register(Registries.BLOCK, new Identifier(mod.metadata().id(), "example_block"), EXAMPLE_BLOCK);
+```
+
+## Adding a BlockItem
+
+Having done that, we can place the block using the `setblock` command, but there isn't an item to place it. So we additionally register a `BlockItem` for the block and add it to the `BUILDING_BLOCKS` item group:
+
+```java
+Registry.register(Registries.ITEM, new Identifier(mod.metadata().id(), "example_block"), new BlockItem(EXAMPLE_BLOCK, new QuiltItemSettings()))
+
+ItemGroupEvents.modifyEntriesEvent(ItemGroups.BUILDING_BLOCKS).register(entries -> {
+	entries.addItem(EXAMPLE_BLOCK.asItem());
+});
+```
+
+For more information on what this does, see the [Creating your first Item](../items/first-item#registering-the-item) Article.
+
+## Adding a Model for the Block
+
+First, we need to create the blockstates file for your Block. This tells the game which `BlockState`s use which models. For an example on using a more complicated `BlockState` setup, see [Adding Redstone functionality to your Block](redstone-interaction). In this case, we only have one `BlockState` and can use this simple JSON:
+
+`assets/simple_block_mod/blockstates/example_block.json`:
+
+```json
+{
+	"variants": {
+		"": {
+			"model": "simple_block_mod:block/example_block"
+		}
+	}
+}
+```
+
+Replace `simple_block_mod` and `example_block` with your mod id and block name. Be sure to replace them in both the file/folder names and the JSON file.
+
+---
+
+Now we can put the model at the path specified by the identifier (in this case `assets/simple_block_mod/models/block/example_block.json`).
+
+```json
+{
+	"parent": "minecraft:block/cube_all",
+	"textures": {
+		"all": "simple_block_mod:block/example_block"
+	}
+}
+```
+
+Replace `simple_block_mod` and `example_block` as before.
+This will use the texture located at `assets/simple_block_mod/textures/block/example_block.png` for all sides of the block.
+
+---
+
+To use this model for the block item too, declare the Block model as a parent in the item model which is located at `assets/simple_block_mod/models/item/example_block.json`:
```

Comment:
```suggestion
Our block item needs a model too, but instead of simply using a texture as we did in the [item tutorial](../items/first-item), we'll declare the Block model as a parent in the item model (located at `assets/simple_block_mod/models/item/example_block.json`). This will render the block item identically to how the block would look in-game.
```
clarification on what this does and why